crypto/property/property_parse.c

```diff
@@ -358,14 +347,13 @@ OSSL_PROPERTY_LIST *ossl_parse_property(OPENSSL_CTX *ctx, const char *defn)
             goto err;
         prop->oper = PROPERTY_OPER_EQ;
         if (prop->name_idx == 0) {
-            PROPerr(PROP_F_OSSL_PARSE_PROPERTY, PROP_R_PARSE_FAILED);
-            ERR_add_error_data(2, "Unknown name HERE-->", start);
+            ERR_raise(ERR_LIB_PROP, PROP_R_PARSE_FAILED,
+                      "Unknown name HERE-->%s", start);
```

Comment:
```"Unknown name \"%s\""``` is much better than ```HERE-->```
In all places where that construct is used.

Reply:
I have kept the existing message. Also, the pointer doesn't show just the name, but everything that follows in the property string. I can't insert a NUL byte in there